frontend/coprs_frontend/coprs/logic/builds_logic.py

```diff
@@ -129,6 +129,18 @@ def get_running_tasks_by_time(cls, start, end):
 
         return result
 
+    @classmethod
+    def get_users_by_time(cls, start, end):
+        """ Returns number of user that submitted at least one build in
+            the period.
+        """
+        result = db.session.query(models.Build.user_id)\
+            .join(models.BuildChroot)\
+            .filter(models.BuildChroot.ended_on > start)\
```

Comment:
This filter seems redundant.  To check that the user is active, it is enough to monitor the "started_on" field.

Reply:
True for the params we are passing. But not true for any time period.
And I basically steal this from the fuction above.